src/protected-api-access-protected-api/README.md

```diff
@@ -0,0 +1,208 @@
+| **Platform type**              | **Web**                                                             | Required value for this sample. <br/> Enables the required and optional settings for the app type.          |
+| **Identifier URI**             | `api://{clientId}`                                                  | Suggested value for this sample. <br/> You must change the client id using the Value shown in Azure portal. |
+| **Expose an API**              | `api://{clientId}/user_impersonation`                               | Create a new delegated permission called user_impersonation. Required value for this sample.                |
+| **API Permissions**            | `https://graph.microsoft.com/User.Read`                             | Create a new delegated permission called for `Microsoft Graph User.Read`. Required value for this sample.   |
+| **Client secret**              | _Value shown in Azure portal_                                       | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).                 |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box or select in the Azure portal.
+
+### 2. Register a client application in your Azure Active Directory (Azure AD)
+
+Second, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the client sample app.
+
+Use the following settings for your app registration:
+
+| App registration <br/> setting | Value for this sample app                                           | Notes                                                                                                       |
+|:------------------------------:|:--------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+| **Name**                       | `active-directory-curl-app`                                         | Suggested value for this sample. <br/> You can change the app name at any time.                             |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**  | Required for this sample. <br/> Support for the Single tenant.                                              |
+| **Platform type**              | **Web**                                                             | Required value for this sample. <br/> Enables the required and optional settings for the app type.          |
+| **API Permissions**            | `api://{clientId}/user_impersonation`                               | Create a new delegated permission called user_impersonation. Required value for this sample.                |
```

Comment:
Why would the client need to create this?  It would be the API's permission that it adds.

Reply:
hopefully we did better now @ckittel, but open to keep iterating till we find the version we like the most


done | addressed from [caf71](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/13/commits/caf712624342176207e6b30b7a670b232576b441)

Reply:
That helps add clarity, IMO. Thanks.